test/e2e/case19_rep_policy_placement_test.go

```diff
@@ -1,241 +0,0 @@
-			plr, metav1.UpdateOptions{},
-		)
-		Expect(err).ToNot(HaveOccurred())
-		plc = utils.GetWithTimeout(
-			clientHubDynamic, gvrPolicy, testNamespace+"."+case19PolicyName, "managed1", false, defaultTimeoutSeconds,
-		)
-		Expect(plc).To(BeNil())
-		plc2 = utils.GetWithTimeout(
-			clientHubDynamic, gvrPolicy, testNamespace+"."+case19PolicyName, "managed2", true, defaultTimeoutSeconds,
-		)
-		Expect(plc2).ToNot(BeNil())
-		afterString := utils.GetMetrics("controller_runtime_reconcile_total",
-			`controller=\"replicated-policy\"`,
-			`,result=\"success\"`,
-		)[0]
-		afterTotal, err := strconv.Atoi(afterString)
-		Expect(err).ShouldNot(HaveOccurred())
-		Expect(afterTotal - beforeTotal).Should(Equal(1))
-	})
-	It("should reconcile twice when managed1 created and managed2 deleted By placementBinding", func() {
-		// Now, connected cluster is managed2
-		By("Get controller_runtime_reconcile_total number")
-		beforeString := utils.GetMetrics("controller_runtime_reconcile_total",
-			`controller=\"replicated-policy\"`,
-			`,result=\"success\"`,
-		)[0]
-		beforeTotal, err := strconv.Atoi(beforeString)
-		Expect(err).ShouldNot(HaveOccurred())
-		By("Patching test-policy-plr with placementDecision of cluster managed1")
-		plr := utils.GetWithTimeout(
-			clientHubDynamic,
-			gvrPlacementDecision,
-			case19PolicyName+"-plr-2",
-			testNamespace,
-			true,
-			defaultTimeoutSeconds,
-		)
-		plr.Object["status"] = utils.GeneratePldStatus(plr.GetName(), plr.GetNamespace(), "managed1")
-		_, err = clientHubDynamic.Resource(gvrPlacementDecision).Namespace(testNamespace).UpdateStatus(
-			context.TODO(),
-			plr, metav1.UpdateOptions{},
-		)
-		Expect(err).ShouldNot(HaveOccurred())
-		By("Patching gvrPlacementBinding placementRef")
-		plb := utils.GetWithTimeout(
-			clientHubDynamic,
-			gvrPlacementBinding,
-			case19PolicyName+"-pb",
-			testNamespace,
-			true,
-			defaultTimeoutSeconds,
-		)
-		plb.Object["placementRef"].(map[string]interface{})["name"] = case19PolicyName + "-plr-2"
-		_, err = clientHubDynamic.Resource(gvrPlacementBinding).Namespace(testNamespace).Update(
-			context.TODO(),
-			plb, metav1.UpdateOptions{},
-		)
-		Expect(err).ToNot(HaveOccurred())
-		By("Check replicate policy changed")
-		plc := utils.GetWithTimeout(
-			clientHubDynamic, gvrPolicy, testNamespace+"."+case19PolicyName, "managed1", true, defaultTimeoutSeconds,
-		)
-		Expect(plc).ToNot(BeNil())
-		plc2 := utils.GetWithTimeout(
-			clientHubDynamic, gvrPolicy, testNamespace+"."+case19PolicyName, "managed2", false, defaultTimeoutSeconds,
-		)
-		Expect(plc2).To(BeNil())
-		afterString := utils.GetMetrics("controller_runtime_reconcile_total",
-			`controller=\"replicated-policy\"`,
-			`,result=\"success\"`,
-		)[0]
-		afterTotal, err := strconv.Atoi(afterString)
-		Expect(err).ShouldNot(HaveOccurred())
-		Expect(afterTotal - beforeTotal).Should(Equal(2))
-	})
-	It("should reconcile twice when managed1 and managed2 are created by placementRule", func() {
```

Comment:
I don't have a strong opinion, but should it just be this test node that gets removed?

Reply:
You mean just remove the test after line 166?

Reply:
As an alternative, yes, but @mprahl indicated the tests don't have a lot of value, which I'm fine with, so removing it is also fine with me.